app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,19 @@
+class FoodsController < ApplicationController
+  def index
+    response = Faraday.get('https://api.nal.usda.gov/fdc/v1/foods/search') do |request|
+      request.params['api_key'] = 'nTNqChgK4a5uwh6kyqJ0B2NCCC9IovJRJT5oLrge'
```

Comment:
😬 😬 careful.. The set up instructions mention the need to run `bundle exec figaro install` so that we have a safe place to store our api keys (applications.yml).